db/migrate/20180403144853_convert_vm_attrs_to_hash.rb

```diff
@@ -0,0 +1,67 @@
+class ConvertVmAttrsToHash < ActiveRecord::Migration[5.1]
+  def up
+    transform :to_h
+  end
+
+  YML_HASH = '!ruby/hash:ActiveSupport::HashWithIndifferentAccess'
+  YML_PARAMS = /\!ruby\/[\w-]+:ActionController::Parameters/
```

Comment:
Looks like we don't need to escape `!` in the regex here and in the reports, sorry for misleading you.